lib/BackendClient.js

```diff
@@ -433,6 +457,41 @@ var BackendClient = (function(){
         return deferred.promise;
     }
 
+    function createRecordedEntry(entryObj)  {
+        let deferred = Q.defer();
+        client.startMultiRequest();
+        // entryServerNode.listAction() parameter
+        let entryServerNodeFilter = new kalturaVO.KalturaEntryServerNodeFilter();
+        entryServerNodeFilter.entryIdEqual = entryObj.entryId;
+        let streamState = '{1:result:objects:0:status}';
```

Comment:
again... we already know the entryServerNodeId... no need to call list again?

Reply:
Not related to entryServerNodeId